lib/jxl/enc_ac_strategy.cc

```diff
@@ -664,22 +666,83 @@ void TryMergeToACSCandidate(AcStrategy::Type acs_raw, size_t bx, size_t by,
       entropy_mul * EstimateEntropy(acs, (bx + cx) * 8, (by + cy) * 8, config,
                                     cmap_factors, block, scratch_space,
                                     quantized);
-
   if (entropy_candidate >= entropy_current) return;
-
   // Accept the candidate.
   for (size_t iy = 0; iy < acs.covered_blocks_y(); iy++) {
     for (size_t ix = 0; ix < acs.covered_blocks_x(); ix++) {
       entropy_estimate[(cy + iy) * 8 + cx + ix] = 0;
       priority[(cy + iy) * 8 + cx + ix] = candidate_priority;
-      ac_strategy->Set(bx + cx + ix, by + cy + iy,
-                       static_cast<AcStrategy::Type>(0));
     }
   }
   ac_strategy->Set(bx + cx, by + cy, acs_raw);
   entropy_estimate[cy * 8 + cx] = entropy_candidate;
 }
 
+void Try16X8And8X16(size_t bx, size_t by, size_t cx, size_t cy,
```

Comment:
This idea could be generalized to 32x16 and 16x32 (and even 64x32 and 32x64). Same code, just some constants become variables.

Reply:
While I naturally agree with that would be an improvement, I'd like to submit this first -- it is already giving an improvement and has been verified independently on discord to improve on our weakest material (anime drawings).

Reply:
Sure. Maybe add a TODO comment?

Reply:
I improved comments on this and added a TODO.